scripts/test-install.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env sh
+
+# Pack the module into a tarball
+npm pack
+tarball=$(realpath puppeteer-*.tgz)
+cd "$(mktemp -d)"
+npm init --yes
```

Comment:
Added this because otherwise you get npm warning you about non existing `package.json` which at first glance looks a bit like an error, but it's not. I think silencing those errors is preferable.

Reply:
If that's the goal, then let's just use `--loglevel silent` for `npm install`?